packages/manager/apps/pci-savings-plan/src/components/Dashboard/Kpis/Kpis.tsx

```diff
@@ -46,14 +49,25 @@ const Kpi = ({
           </OdsText>
         </OdsTooltip>
       </OdsText>
-      <span className="text-[20px] font-bold mb-0 text-[#0050D7]">{value}</span>
+      <span className="text-[20px] font-bold mb-0 text-[#0050D7]">
+        {value !== null ? value.toFixed(2) : t('dashboard_kpis_not_available')}
+      </span>
     </div>
   );
 };
 
-const Kpis = ({ isLoading }: Readonly<{ isLoading: boolean }>) => {
+const Kpis = ({
+  isLoading,
+  consumption,
+}: Readonly<{
+  isLoading: boolean;
+  consumption: SavingsPlanConsumption;
+}>) => {
   const { t } = useTranslation('dashboard');
-  const data = [
+  const computedUsagePercent = useMemo(() => {
+    return consumption ? calculateAverageUsage(consumption.flavors) : null;
+  }, [consumption]);
```

Comment:
You can keep ES6 syntax :)
```suggestion
  const computedUsagePercent = useMemo(() => consumption ? calculateAverageUsage(consumption.flavors) : null, [consumption]);
```